include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -217,9 +217,9 @@ class argument_parser
                     option_spec const & spec = option_spec::DEFAULT,
                     validator_type validator = validator_type{}) // copy to bind rvalues
     {
-        if (id_exists(short_id))
+        if (id_exists(short_id) && short_id != '\0')
```

Comment:
Please remove this again and instead change the `id_exists` function : 
(This way will require only 2 code line changes instead of 4 and I think it belongs to the functionalty of `id_exists`)
example for long_id
```cpp
    bool id_exists(std::string const & long_id)
    {
        if (long_id.empty())
            return false;
        return(!(used_option_ids.insert(long_id)).second);
    }

```